src/Core/src/Platform/Windows/WrapperView.cs

```diff
@@ -184,6 +184,28 @@ void OnChildSizeChanged(object sender, SizeChangedEventArgs e)
 			UpdateShadowAsync().FireAndForget(IPlatformApplication.Current?.Services?.CreateLogger(nameof(WrapperView)));
 		}
 
+		protected override global::Windows.Foundation.Size ArrangeOverride(global::Windows.Foundation.Size finalSize)
```

Comment:
Actually I debug it manually , while without fix, the Wrapperview.updateclip is updated properly with visual.clip in Wrapperview... What i was observed is when updating the clip in sizechanged method in sample level ... Contentpanel Visual.Clip (border) is updated lastly may cause the confusion with the wrapperview visual.clip .... byt using the child.Layoutupdated changes in wrapperview working fine ... but iam not sure that is proper solution --